app/views/unattended/provisioning_templates/snippet/first_boot_setup_service.erb

```diff
@@ -0,0 +1,20 @@
+<%#
+kind: snippet
+name: first_boot_setup_service
+model: ProvisioningTemplate
+snippet: true
+description: |
+  Post replacement service
+-%>
+[Unit]
+Description=Initial setup callback
+Wants=network-online.target
+After=network-online.target
+
+[Service]
+ExecStart=/bin/bash /root/fist_boot_setup_script.sh
```

Comment:
```suggestion
ExecStart=/bin/bash /root/first_boot_setup_script.sh
```

Fist booting a machine would be a bit aggressive :)

Reply:
That indeed would be :D